### ISSUE TITLE
google_checks.xml: false negative 3.3.1.1 No module imports

### ISSUE THREAD

**Issue by Atharv3221** [checkstyle/checkstyle#21445]
Child Issue of https://github.com/checkstyle/checkstyle/issues/21428

I have downloaded the latest checkstyle from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the cli and showed it below, as cli describes the problem better than 1,000 words

[3.3.1.1 No module imports](https://google.github.io/styleguide/javaguide.html#s3.3.1.1-module-imports)

<img width="803" height="171" alt="Image" src="https://github.com/user-attachments/assets/7343298b-8765-4833-b8b6-fd2356375f9b" />

## Module Import
```java
atharv@atharv-IdeaPad-3-15IIL05:~/test$ cat Test.java
import module java.base;

/** Some javadoc. */
public class Test {
  void method() {
  }
}
```
```
atharv@atharv-IdeaPad-3-15IIL05:~/test$ java -jar checkstyle-14.1.0-all.jar -c /google_checks.xml Test.java
Starting audit...
Audit done.
```
Expected:
```
Starting audit...
[ERROR] /home/atharv/test/Test.java:1:1: Using the 'import module' form of import should be avoided - java.base. [AvoidModuleImport]
Audit done.
Checkstyle ends with 1 errors.
```

**Comment by romani** [comment]
We should do this by xpath match module

**Comment by NianJiuZst** [comment]
I am on it.

**Comment by Atharv3221** [comment]
@NianJiuZst please work on another issue

**Comment by Atharv3221** [comment]
I am working on it
